### ISSUE TITLE
Unable to load files with some special characters

### ISSUE THREAD

**Issue by blackbird-91** [atteneder/glTFast#79]
When a glb file has a '+' character in it, like "file_a + file_b.glb", the co-routine to load the file fails to read the file. Since windows and mac allows some special characters in filenames, I can't have control on the file names themselves. Is there a way to fix this issue?

```
IEnumerator LoadRoutine( string url, bool gltfBinary ) {
            
            var download = downloadProvider.Request(url);
            yield return download;

            if(download.success) {
                if(gltfBinary) {
                    LoadGlb(download.data,url);
                } else {
                    LoadGltf(download.text,url);
                }
                yield return LoadContent();
            } else {
                Debug.LogErrorFormat("{0} {1}",download.error,url);
                loadingError=true;
            }

            DisposeVolatileData();
            OnLoadComplete(!loadingError);
        }
```

**Comment by atteneder** [comment]
Hi,

sorry for the late answer. glTFast uses regular UnityWebRequest under the hood. Everything they can load, glTFast can load. A quick workaround is to escape the `+` like this:

`url = url.Replace("+","%2B");`

This worked here also on local paths (starting with `file://`). There's also the more generic [`UnityWebRequest.EscapeURL`](https://docs.unity3d.com/ScriptReference/Networking.UnityWebRequest.EscapeURL.html), but make sure you don't escape the full path (as it will escape directory separators as well).

Now this won't help in case any references external file in the glTF (buffer or texture) has a `+`in its URL. I'd have to fix this inside of glTFast.

hth

**Comment by atteneder** [comment]
Regarding internal URIs, the way I interpret the [glTF URI specification](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#uris) I think it's not required to support special characters, but optional. Thus any content-creator creating such URIs has to be aware that his files won't be widely portable.

**Comment by cvetelinast** [comment]
Hi, thank you for your awesome library! You said that `UnityWebRequest.Get(url)` doesn't support special characters, but then you added a commit about that, should that commit handle this scenario? I can't really make it work.
I have the file `A+B.gltf ` with buffer `A+B.bin`.
I tried the workarounds that you suggested, for example, I tried to make the field `url` in GLTFAsset.cs `A%2BB.gltf` but the file is still not found. If I rename the gltf, it still cannot find the bin.
